packages/protocol-kit/src/types/safeConfig.ts

```diff
@@ -40,7 +41,7 @@ type SafeConfigWithPredictedSafeProps = {
 
 export type SafeConfigProps = {
   provider: SafeProviderConfig['provider']
-  signer?: SafeProviderConfig['signer']
+  signer?: string | passkeyArgType
```

Comment:
Why this change? SafeProviderConfig signer should be the correct one right?

Reply:
true! I restored it in the other PR:  https://github.com/safe-global/safe-core-sdk/pull/841/files#diff-d7c347ab5fed07174eb38d1a9731c90191e250ca1afabe36c35924940823a8cbR44